### PR TITLE
Update the NextJS guide #545

### PR DESCRIPTION
**Pull request by pmi**

### CODE REVIEW

File: docs/layouts.adoc [hunk 1]
```diff
@@ -30,6 +30,9 @@ NOTE: Just like a page, layouts may declare named regions.
 +
 . **Add the preview helper**
 +
+IMPORTANT: This is only required if you are using <<library#, Next.js library>> !
+If you are using <<preview#, Next.js app>> you can skip this action.
++
```
Comment (poi33) [inline docs/layouts.adoc:35]: Not a fan of this. 
We assume that they are using the preview app, so make a note/section on what need to be added if you are using the library

**Review by poi33 — CHANGES_REQUESTED** [review]

File: docs/library.adoc [hunk 2]
```diff
@@ -0,0 +1,174 @@
+= Next.js library
+:toc: right
+:imagesdir: media/
+
+In this chapter, you'll enable Content Studio to load the preview from Next using the Next.js integration library.
+
+IMPORTANT: This is an advanced approach requiring additional configuration !
```
Comment (poi33) [inline docs/library.adoc:7]:
```suggestion
IMPORTANT: This is an advanced approach requiring additional configuration!
```

File: docs/library.adoc [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
++
+image:edit-site.png[title="Select site item and click edit",width=650px]
++
+Configure the "Headless Movie DB" application by clicking the pencil icon.
++
+image:edit-site-app.png[title="Open app config by pressing pencil icon",width=597px]
++
+Finally - if you did not change the API token in the Next app, use these values: `http://localhost:3000[http://localhost:3000^]`, and `mySecretKey`.
++
+image:edit-hmdb-app.png[title="Form with fields for server url and nextjs token",width=771px]
```
Comment (poi33) [inline docs/library.adoc:78]: We don't mention the app config method? If possible please avoid undocumented features

**Review by poi33 — CHANGES_REQUESTED** [review]

File: docs/library.adoc (hunk 3 above)
Comment (pmi) [reply docs/library.adoc:78]: Added a mention about app config file

**Pull request merged**